### DISCUSSION TITLE
A gyvi?

### DISCUSSION THREAD

**Discussion by Othnus** [WeDish/wedish_backend#119] [General]
Ar gyvi? Kaip sekasi jums? Ar užsikabinot su programivu? Gal kam įdomu būtu susitikti, ar susiskambinti paplepinti apie savo kelionę?
@WeDish/founders

**Comment by vincisda** [comment]
Po kursu priėmė i Roboticsu (RPA)komanda SEB banke kaip Automation Developer. Ten iki šiol ir dirbu. Nieko su Python daryt neteko ir darbe neprireikė, tai sėkmingai viska pamirsau. Didžiausias pasiekimas su VBA parašyt code stage kad rekursijos būdu iš nestinto JSON ištraukti kolekcija pagal value. Tai tiek iš manes.
